src/hardware/serialport/serialmouse.cpp

```diff
@@ -0,0 +1,493 @@
+/*
+ *  Copyright (C) 2022       The DOSBox Staging Team
+ *  Copyright (C) 2002-2021  The DOSBox Team
```

Comment:
This looks like a whole new file authored by just you.

Was some of it written by the The DOSBox Team members during those prior years?

I want authorship to be as correct as possible. If we can ascribe authorship to the upstream team members who contributed to this file, then we should also add them as co-authors in the Git commit (the commit that adds this file).

If you do a `git log | grep '@vogons'`, you can get author IDs for upstream members :+1:

Reply:
Alternatively, feel free to use yourself as the sole copyright author. Whatever is the most factual.

Reply:
I'm not the sole author - the file was taken from DOSBox X and heavily reworked/extended by me. Thus, I have replaced the original note:
```
/* Microsoft Serial Mouse compatible emulation.
 * Written by Jonathan Campbell */
```
with:
```
// Microsoft Serial Mouse emulation originally written by Jonathan Campbell
// Wheel, Logitech, and Mouse Systems mice added by Roman Standzikowski (FeralChild64)
```
And this is enough :)